cleanlab/count.py

```diff
@@ -734,6 +761,9 @@ def estimate_py_noise_matrices_and_cv_pred_proba(
         Set the default state of the random number generator used to split
         the cross-validated folds. If None, uses np.random current random state.
 
+    clf_kwargs : dict, default = {}
+          Optional keyword arguments to pass into `clf` fit() method.
```

Comment:
Can change indentation to 2 spaces

Reply:
Thanks, done. We'll need to standardize the docstring indentation anyway across the methods/files, but that'll be saved for a separate big PR.